sitemap.js

```diff
@@ -82,6 +89,35 @@ see https://www.sitemaps.org/protocol.html for more information`
     );
   }
 
+  // images
+  const _imagesError = `images ${JSON.stringify(
+    images
+  )} was not valid. An array with image locations like
+
+[{
+  loc: 'https://example.com/test/my-image.jpg'
+},{
+  loc: 'https://example.com/test/another-image.png'
+}]
+
+was expected.
+
+see https://support.google.com/webmasters/answer/178636?hl=en for more information.`;
+
+  if (images !== undefined) {
+    if (!(images instanceof Array)) {
```

Comment:
Any reason for not using `Array.isArray()`?

Reply:
seems like that's just being consistent with my code style earlier. Will update separately